src/main/webapp/user-page.html

```diff
@@ -41,6 +41,13 @@ <h1 id="page-title">User Page</h1>
     <hr/>
 
     <div id="message-container">Loading...</div>
-
+    <div id="about-me-container">Loading...</div>
+      <div id="about-me-form" class="hidden">
+        <form action ="\about" method="POST">
```

Comment:
Replace '\' with '/' since '\' is not a valid URL path character.

Reply:
Implemented in 8254fef99048d0b21c32dbdd58ce3c4bcba087e3